src/main/java/Duke/Tasks/TaskList.java

```diff
@@ -0,0 +1,146 @@
+package Duke.Tasks;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import Duke.Exception.DataCorruptedException;
+import Duke.Tasks.TaskType;
+
+public class TaskList {
+    private final String FILE_PATH = "./data/data.md";
+    private ArrayList<Task> inputLists = new ArrayList<Task>();
+
+    public void initialiseTaskFromFile()
+            throws FileNotFoundException, DataCorruptedException {
+        File f = new File(FILE_PATH);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String[] input = s.nextLine().split("\\|");
+            if (input[1].strip().equals("Task Type") || input[1].strip().equals("----------")) {
```

Comment:
For both lines, you might like to consider relocating the values you are comparing with to variables instead, so as to eliminate magic literals.